<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Box</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="0" column="0">
     <widget class="QTableView" name="tableView2"/>
    </item>
   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewCrate"/>
    <addaction name="separator"/>
    <addaction name="actionNewBox"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
   <property name="title">
    <string>Run</string>
   </property>
<!--    <addaction name="actionRun"/>-->
    <addaction name="actionRunMul"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
<!--    <addaction name="actionShow"/>-->
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
<!--   <addaction name="menuShow"/>-->
 </widget>




  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewCrate">
   <property name="text">
    <string>Add Crate</string>
   </property>
  </action>
  <action name="actionNewBox">
   <property name="text">
    <string>Add Box</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
   <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
   <action name="actionRunMul">
   <property name="text">
    <string>Run Multiple</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Boxes List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
